src/sentry/templates/sentry/emails/error.html

```diff
@@ -38,7 +38,21 @@
   </div>
 {% endblock %}
 
-{% block main %}
+{% block content %}
+{% if group.project.flags.has_transactions %}
```

Comment:
@Neo-Zhixing We only want to show this block if the project doesn't have transactions, this looks like the opposite of that

Reply:
Great catch! Thanks!